batch/batch/worker/worker.py

```diff
@@ -2265,7 +2247,8 @@ async def create_container_and_connect(
                 except (FileNotFoundError, ConnectionRefusedError) as err:
                     attempts += 1
                     if attempts == 240:
-                        jvm_output = await container.container.get_log() or ''
+                        # NOTE we assume that the JVM logs hail emits will be valid utf-8
+                        jvm_output = (await fs.read(container.container.log_path)).decode('utf-8')
```

Comment:
Can we guarantee there is always a file existing at `container.container.log_path`?

Reply:
Hm maybe not here, I'll protect against this